compiler/rustc_data_structures/src/graph/iterate/mod.rs

```diff
@@ -87,8 +87,7 @@ where
 }
 
 /// Allows searches to terminate early with a value.
-// FIXME (#75744): remove the alias once the generics are in a better order and `C=()`.
-pub type ControlFlow<T> = std::ops::ControlFlow<(), T>;
+pub use std::ops::ControlFlow;
```

Comment:
Re-exporting a standard library type is somewhat unusual, could you change the use sites to import `std::ops::ControlFlow` instead?